src/platform_impl/web/event_loop/runner.rs

```diff
@@ -158,8 +158,11 @@ impl<T: 'static> Shared<T> {
     }
 
     pub fn init(&self) {
-        let start_cause = Event::NewEvents(StartCause::Init);
-        self.run_until_cleared(iter::once(start_cause));
+        // NB: For consistency all platforms must emit a 'resumed' event even though web
+        // applications don't themselves have a formal suspend/resume lifecycle.
+        let init_events =
+            iter::once(Event::NewEvents(StartCause::Init)).chain(iter::once(Event::Resumed));
```

Comment:
Rust nowadays implements `IntoIter` directly on static-sized arrays, right? Meaning you could write this as `[Event::NewEvents(StartCause::Init), Event::Resumed].into_iter()`.

Reply:
Yeah, that'd sounds good - tbh, I think I just made that change on autopilot after I saw that code was already using `.chain()` for linking events but yeah in this case it's kinda silly to wrap two iter::once iters.